src/main/java/io/confluent/connect/hdfs/parquet/ParquetRecordWriterProvider.java

```diff
@@ -74,7 +78,15 @@ public void write(SinkRecord record) {
                 true,
                 conf.getHadoopConfiguration()
             );
+            log.debug("Opened record writer for: {}", filename);
           } catch (IOException e) {
+            log.debug(
+                "Error creating AvroParquetWriter for file '{}', {}, and schema {}: ",
```

Comment:
Would it make sense to use `AvroParquetWriter.class.getSimpleName()` here in case this changes in the future and the log message is left behind?